pkgs/applications/networking/cluster/argo-rollouts/default.nix

```diff
@@ -0,0 +1,41 @@
+{ buildGoModule, lib, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "argo-rollouts";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "argoproj";
+    repo = "argo-rollouts";
+    rev = "v${version}";
+    sha256 = "0qb1wbv3razwhqsv972ywfazaq73y83iw6f6qdjcbwwfwsybig21";
+  };
+
+  vendorSha256 = "1rmw80lvs9qj2d195rvhzz8xrbx71dirv1c0grc113d18q609qwj";
+
+  # Disable tests since some test fail because of missing test data
+  doCheck = false;
+
+  runVend = true;
```

Comment:
https://github.com/NixOS/nixpkgs/blob/62aa9182375579b3e72b13e6cd98fcb5d36c6315/pkgs/development/go-modules/generic/default.nix#L58

Check if it actually needs `proxyVendor` though, usually shouldn't be necessary.

Reply:
How should I check the `proxyVendor` ?

Reply:
Regenerate the `vendorSha256` and see if it builds the `go-modules` and package, in this case it's not needed.

```diff
diff --git a/pkgs/applications/networking/cluster/argo-rollouts/default.nix b/pkgs/applications/networking/cluster/argo-rollouts/default.nix
index 29a7111e310..8127904023c 100644
--- a/pkgs/applications/networking/cluster/argo-rollouts/default.nix
+++ b/pkgs/applications/networking/cluster/argo-rollouts/default.nix
@@ -11,13 +11,11 @@ buildGoModule rec {
     sha256 = "0qb1wbv3razwhqsv972ywfazaq73y83iw6f6qdjcbwwfwsybig21";
   };
 
-  vendorSha256 = "1rmw80lvs9qj2d195rvhzz8xrbx71dirv1c0grc113d18q609qwj";
+  vendorSha256 = "00ic1nn3wgg495x2170ik1d1cha20b4w89j9jclq8p0b3nndv0c0";
 
   # Disable tests since some test fail because of missing test data
   doCheck = false;
 
-  runVend = true;
-
   subPackages = [ "cmd/rollouts-controller" "cmd/kubectl-argo-rollouts" ];
 
   meta = with lib; {
```

Reply:
Ah, I see. Thanks!

> in this case it's not needed.

Cool, I'm marking this comment as resolved then. :-)

Reply:
`runVend` still needs to be removed.

Reply:
Fixed, thanks!